src/Components/Coins/CoinsButton.js

```diff
@@ -20,7 +20,7 @@ export default function CoinsButton() {
 
   return (
     <div id="coinsButtonContainer">
-      <GeneralButton id="coinsButton" uppercased outlined onButtonClick={() => onCoinsButtonClick(isCoinsTabOpen)}>
+      <GeneralButton id="coinsButton" uppercased outlined onButtonClick={() => onCoinsButtonClick()}>
```

Comment:
good catch, as I said.
if you're here - could you also change `() => onCoinsButtonClick()` to `onCoinsButtonClick`?

Reply:
didn't get you here.